x-pack/examples/alerting_example/server/plugin.ts

```diff
@@ -7,16 +7,21 @@
 
 import { Plugin, CoreSetup } from 'kibana/server';
 import { i18n } from '@kbn/i18n';
-import { DEFAULT_APP_CATEGORIES } from '../../../../src/core/server';
+// import { DEFAULT_APP_CATEGORIES } from '../../../../src/core/server/';
```

Comment:
Any reason to keep these commented-out sections?

Reply:
removed